components/dash-core-components/src/components/Dropdown.react.js

```diff
@@ -145,6 +145,11 @@ Dropdown.propTypes = {
      */
     disabled: PropTypes.bool,
 
+    /**
+     * If false, the menu of the dropdown will not close once a value is selected.
+     */
+    closeMenuOnSelect: PropTypes.bool,
```

Comment:
Props are `snake_case`

Reply:
Oh I guess these are given as is to the dropdown, in that case it's good in camelCase for consistency.